### PR TITLE
Star wars skin

### PR DESCRIPTION
**Pull request by epeach**
Report from curriculum that 'goals' in Star Wars Playlab projects built through levelbuilder were showing up getting 404s. Problem: Star Wars legacy projects use goalOverride to specify images, but there was no place to do that in levelbuilder. Resolution: Adding a field in levelbuilder to add goal overrides.

Note: Some .level files have null goal-overrides. This was messing with the new system of either specifying overrides or not having that level data at all. I manually deleted the goal override in these levels. There should be no impact of doing so as they were previously null.

Screenshot of error:
404_star_wars

Screenshot of added field in levelbuilder:
Screenshot from 2020-03-05 11-32-32

## Links

- [spec]()
- [jira]()

## Testing story

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by jmkulwik — APPROVED** [review]
Nice fix!

File: dashboard/app/models/levels/blockly.rb [hunk 1]
```diff
@@ -609,4 +610,18 @@ def localized_shared_blocks(level_objects)
...
     level_objects_copy
   end
+
+  def update_goal_override
+    if goal_override.present? && goal_override.is_a?(String)
+      self.goal_override = JSON.parse(goal_override)
+    end
+  end
+
+  def self.goal_override
```
Comment (maddiedierker) [inline dashboard/app/models/levels/blockly.rb]: we aren't currently calling this method anywhere -- i think we want to call this for any Blockly level where `self.goal_override == nil` -- is that right? (making sure i understand before giving add'l guidance)

File: dashboard/app/models/levels/blockly.rb [hunk 2]
```diff
@@ -609,4 +610,18 @@ def localized_shared_blocks(level_objects)
     end
     level_objects_copy
   end
+
+  def update_goal_override
+    if goal_override.present? && goal_override.is_a?(String)
```
Comment (maddiedierker) [inline dashboard/app/models/levels/blockly.rb]: super nit: we can check for presence and type at the same time--
```suggestion
    if goal_override&.is_a?(String)
```

File: dashboard/app/models/levels/blockly.rb (hunk 1 above)
Comment (epeach) [reply dashboard/app/models/levels/blockly.rb]: I actually lifted this from StudioEc and now that I'm looking at it, I'm realizing that we don't want to call this - we only want to add a goalAnimation is the levelbuilder specifies one. Deleting . . .

**Comment by epeach** [comment]
@madelynkasula PTAL! Thanks!

**Review by maddiedierker — APPROVED** [review]
ship it!

bb8

**Pull request merged**